warframe/worldstate/models/void_trader.py

```diff
@@ -19,28 +19,12 @@ class InventoryItem(WorldstateObject):
     "The cost of credits"
 
 
-class VoidTrader(SingleQueryModel):
+class VoidTrader(SingleQueryModel, TimedEvent):
     __endpoint__ = "/voidTrader"
 
     # required
-    start_string: str
-    "Short-time-formatted duration string of the arrival of the Void Trader"
-
-    active: bool
-    "Whether the Void Trader is currently active"
-
     location: str
     "The Relay on which the Void Trader is on"
 
     inventory: List[InventoryItem]
     "The Void Trader's inventory"
-
-    end_string: str
-    "Short-time-formatted duration string of the department of the Void Trader"
-
-    # optional
-    arrival: Optional[datetime] = field(name="activation", default=None)
-    "The time the Void Trader arrived"
-
-    department: Optional[datetime] = field(name="expiry", default=None)
-    "The time the Void Trader departs"
```

Comment:
this is bad English. if you want the equivalent of `arrival`, use `departure`. this is why i used `activation` and `expiry`.... i also don't think you should change this language so much, it will make it harder for anyone who is already familiar with the worldstate to use this

Reply:
I deleted it, it's now just activation/expiry